HW2/P3/P3.txt

```diff
@@ -0,0 +1,32 @@
+Without Instruction-level Parallelism
+
+1 thread:
+  1074.656613 Million Complex FMAs in 6.87227201462 seconds,
+  156.375738724 million Complex FMAs / second
+
+2 threads:
+  1074.656613 Million Complex FMAs in 3.39673495293 seconds,
+  316.379295969 million Complex FMAs / second
+
+4 threads:
+  1074.656613 Million Complex FMAs in 1.82714009285 seconds,
+  588.163226895 million Complex FMAs / second
+
+
+With Instruction-level Parallelism
+
+1 thread:
+  11560639453.7 Million Complex FMAs in 2.80734610558 seconds,
+  4117995793.52 million Complex FMAs / second
+
+2 threads:
+  11560639453.7 Million Complex FMAs in 1.43554806709 seconds,
+  8053119027.31 million Complex FMAs / second
+
+4 threads:
+  11560639453.7 Million Complex FMAs in 0.838935852051 seconds,
+  13780123266.2 million Complex FMAs / second
+
+Something is clearly awry with my instruction-level parallelism,
+because 11560639453.7 million is hardly feasible, but if you compare
+the FMA/sec, you'll see it doubles each time from 1 to 2 and 2 to 4 threads. 
```

Comment:
Hmm definitely seems like something is going on here. Did you end up with reasonable outputs (i.e., compared to to the original code)?